tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -99,6 +100,13 @@ def get_ad_config_param_vals(
     }
 
 
+def get_fsx_config_param_vals(fsx_factory, svm_factory):
```

Comment:
Minor: In this way the creation of the two types of file systems does not go in parallel, right?
Is it possible to create the two file systems in parallel to save time?

Reply:
They are not parallel. To make it parallel, we have to refactor the code.